src/scripts/ViewsGetAllViews.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
```

Comment:
If this comment is made with `//` then it will be stripped prior to base64 encoding. The regex to strip multiline comments is a lot harder to do.